playwright/src/utilities/utilities.ts

```diff
@@ -28,9 +28,15 @@ function randomChoice(list: ElementHandle[]): ElementHandle {
   return list[option]
 }
 
-function randomNumber(count: number) {
-  const randomnum = Math.floor(Math.random() * count)
-  return randomnum
+function randomNum(count: number, excludenum?: number) {
```

Comment:
At some later point I will work on making the `excludenum` parameter an array so that more than 2 highlights can be made on a page